models/PyCryptoBot.py

```diff
@@ -1052,18 +1052,18 @@ def __init__(self, exchange='coinbasepro', filename='config.json'):
                 self.simstartdate = None
 
         if args.sellupperpcnt != None:
-            if isinstance(args.sellupperpcnt, int):
-                if args.sellupperpcnt > 0 and args.sellupperpcnt <= 100:
+            if isinstance(args.sellupperpcnt, (int,str)):
+                if float(args.sellupperpcnt) > 0 and float(args.sellupperpcnt) <= 100:
                     self.sell_upper_pcnt = float(args.sellupperpcnt)
 
         if args.selllowerpcnt != None:
-            if isinstance(args.selllowerpcnt, int):
-                if args.selllowerpcnt >= -100 and args.selllowerpcnt < 0:
+            if isinstance(args.selllowerpcnt, (int,str)):
+                if float(args.selllowerpcnt) >= -100 and float(args.selllowerpcnt) < 0:
                     self.sell_lower_pcnt = float(args.selllowerpcnt)
 
         if args.trailingstoploss != None:
-            if isinstance(args.trailingstoploss, int):
-                if args.trailingstoploss >= -100 and args.trailingstoploss < 0:
+            if isinstance(args.trailingstoploss, (int,str)):
```

Comment:
Same here... please check for "int" for "float" types. If you enter a string and it tries to cast it as a float below it will crash.